Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsFragmentPresenter.java

```diff
@@ -633,6 +635,9 @@ private void addWiiSettings(ArrayList<SettingsItem> sl)
             R.string.wiimote_scanning, R.string.wiimote_scanning_description));
     sl.add(new SwitchSetting(mContext, BooleanSetting.MAIN_WIIMOTE_ENABLE_SPEAKER,
             R.string.wiimote_speaker, R.string.wiimote_speaker_description));
+
+    sl.add(new SwitchSetting(mContext, BooleanSetting.MAIN_EMULATE_SKYLANDER_PORTAL,
+            R.string.emulate_skylander_portal, 0));
```

Comment:
Please add a heading here, otherwise it looks like this is part of the Wii Remote settings. ("Emulated USB Devices" or something similar would probably work well.)